src/modules/blockchainApplication/explore/api/index.js

```diff
@@ -1,27 +1,2 @@
-/* istanbul ignore file */
-import http from 'src/utils/api/http';
-import { HTTP_PREFIX } from 'src/const/httpCodes';
-
-const httpPaths = {
-  blockchainAppsStatistics: `${HTTP_PREFIX}/blockchain/apps/statistics`,
-};
-
-/**
- * Retrieves sidechains statistics information
- *
- * @param {Object} data
- * @param {Object} data.network The network config from the Redux store
- *
- * @returns {Promise}
- */
-// eslint-disable-next-line import/prefer-default-export
-export const getStatistics = ({
-  network,
-  params = {},
-  baseUrl,
-}) => http({
-  path: httpPaths.blockchainAppsStatistics,
-  network,
-  params,
-  baseUrl,
-});
+export * from './applicationExplore';
+export * from './applicationStatistics';
```

Comment:
I don't think that creating 3 files for just one function is good architecture. I think decoupling files should be done when needed (Separation of concerns), but not for nesting exports and creating single function files, that results on an over-scaffolded file structure.

Reply:
so usually is the best way its have separate a file for each utils so over time they don't get fat and ugly, 
in this spacial case because we mock the HTTP and it has difference between them its needs to be split into two file

Reply:
That I agree, but we did not reach that point and from requirements does not look we will. If files get too large thats why we hace a lint rule of maximum 300 lines. I agree with reducing the amount of lines if need, but right now for me this is counterproductive decoupling. We will end up spending way too much time creating files for just 1 function for no gain. I don't see why mocking HTTP call affects the architecture, specially because it is mocked in HOC component level and not on api util

Reply:
here is the reason on the test file 

application list
```
jest.mock('src/utils/api/http', () =>
  jest.fn().mockImplementation(() => Promise.resolve({ data: [{ type: 0 }] })));
```

application statics
```

jest.mock('src/utils/api/http', () =>
  jest.fn().mockImplementation(() => Promise.resolve({
    data: {
      registered: 2503,
      active: 2328,
      terminated: 35,
      totalSupplyLSK: '5000000',
      stakedLSK: '3000000',
      inflationRate: '4.50',
    },
  })));

```

Reply:
That is an unit test mock. I was talking about mocking in runtime. I don't see neither how mocking HTTP module affects file structure still